dspace-spring-rest/src/main/java/org/dspace/app/rest/link/relation/RelationshipResourceWrapperHalLinkFactory.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.link.relation;
+
+import java.util.LinkedList;
+
+import org.dspace.app.rest.RelationshipRestController;
+import org.dspace.app.rest.link.HalLinkFactory;
+import org.dspace.app.rest.model.RelationshipRest;
+import org.dspace.app.rest.model.RelationshipRestWrapper;
+import org.dspace.app.rest.model.hateoas.EmbeddedPage;
+import org.dspace.app.rest.model.hateoas.RelationshipResourceWrapper;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+import org.springframework.hateoas.Link;
+import org.springframework.stereotype.Component;
+import org.springframework.web.util.UriComponentsBuilder;
+
+@Component
+public class RelationshipResourceWrapperHalLinkFactory
+    extends HalLinkFactory<RelationshipResourceWrapper, RelationshipRestController> {
+    protected void addLinks(RelationshipResourceWrapper halResource, Pageable pageable, LinkedList<Link> list)
```

Comment:
Javadocs please

Reply:
This @override annotation has been included because it's part of the HalLinkFactory